Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scripts/BoundaryVisualizationDemo.cs

```diff
@@ -0,0 +1,134 @@
+
+        /// <summary>
+        /// The material used to display indicators that are outside of the boundary geometry.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("Material used to display indicators that are outside of the boundary geometry.")]
+        private Material outOfBoundsMaterial = null;
+
+        /// <summary>
+        /// The material used to display the inscribed rectangle and the indicators that are within it.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("Material used to display the inscribed rectangle and the indicators that are within it.")]
+        private Material inscribedRectangleMaterial = null;
+
+        /// <summary>
+        /// Boundary system implementation.
+        /// </summary
+        private IMixedRealityBoundarySystem boundaryManager = null;
+        private IMixedRealityBoundarySystem BoundaryManager => boundaryManager ?? (boundaryManager = MixedRealityManager.Instance.GetManager<IMixedRealityBoundarySystem>());
+
+        private void Start()
+        {
+            AddQuad();
+            AddIndicators();
+        }
+
+        /// <summary>
+        /// Displays the boundary as a quad primitive.
+        /// </summary>
+        private void AddQuad()
+        {
+            // Get the rectangular bounds.
+            Vector2 center = EdgeUtilities.InvalidPoint;
+            float angle = 0f;
+            float width = 0f;
+            float height = 0f;
+            if ((BoundaryManager == null) || !BoundaryManager.TryGetRectangularBoundsParams(out center, out angle, out width, out height))
+            {
+                // No rectangular bounds, therefore do not render the quad.
+                return;
+            }
+            
+            // Render the rectangular bounds.
+            if (EdgeUtilities.IsValidPoint(center))
+            {
+                GameObject quad = GameObject.CreatePrimitive(PrimitiveType.Quad);
+                quad.transform.SetParent(transform);
+                quad.transform.Translate(new Vector3(center.x, 0.005f, center.y)); // Add fudge factor to avoid z-fighting
+                quad.transform.Rotate(new Vector3(90, -angle, 0));
+                quad.transform.localScale = new Vector3(width, height, 1.0f);
+                quad.GetComponent<Renderer>().sharedMaterial = inscribedRectangleMaterial;
+            }
+        }
+
+        /// <summary>
+        /// Displays the boundary as an array of spheres where spheres in the
+        /// bounds are a different color.
+        /// </summary>
+        private void AddIndicators()
+        {
+            // Get the rectangular bounds.
+            Vector2 centerRect = EdgeUtilities.InvalidPoint;
+            float angleRect = 0f;
+            float widthRect = 0f;
+            float heightRect = 0f;
+            if ((BoundaryManager == null) || !BoundaryManager.TryGetRectangularBoundsParams(out centerRect, out angleRect, out widthRect, out heightRect))
```

Comment:
Same here, This should be the first thing in the call.

Reply:
they are out params to the TryGet function... they must be defined.